packages/forms/src/lib/mask/README.md

```diff
@@ -0,0 +1,43 @@
+# @acpaas-ui/ngx-components/forms
+
+With the @acpaas-ui mask module you can mask input fields.
```

Comment:
I would give it a bit more info, like mentioning that it actually is a wrapper of the `RobinHerbots/Inputmask package.`
Have a look at the original documentation for some inspiration.